inst/examples/dashboard/examples.R

```diff
@@ -1,1706 +1,62 @@
-        list(
-          name = "Home",
-          expandAriaLabel = "Expand Home section",
-          collapseAriaLabel = "Collapse Home section",
-          links = list(
-            list(
-              name = "Activity",
-              url = "http://msn.com",
-              key = "key1",
-              target = "_blank"
-            ),
-            list(
-              name = "MSN",
-              url = "http://msn.com",
-              disabled = TRUE,
-              key = "key2",
-              target = "_blank"
-            )
-          ),
-          isExpanded = TRUE
-        ),
-        list(
-          name = "Documents",
-          url = "http://example.com",
-          key = "key3",
-          isExpanded = TRUE
-        ),
-        list(
-          name = "Pages",
-          url = "http://msn.com",
-          key = "key4"
-        ),
-        list(
-          name = "Notebook",
-          url = "http://msn.com",
-          key = "key5",
-          disabled = TRUE
-        ),
-        list(
-          name = "Communication and Media",
-          url = "http://msn.com",
-          key = "key6"
-        ),
-        list(
-          name = "News",
-          url = "http://cnn.com",
-          icon = "News",
-          key = "key7",
-          target = "_blank",
-          iconProps = list(
-            iconName = "News",
-            styles = list(
-              root = list(
-                fontSize = 20,
-                color = "#106ebe"
-              )
-            )
-          )
-        )
-      )
-    )
-  )
-
-  list(
-    ui = Nav(
-      groups = link_groups,
-      selectedKey = "key1",
-      styles = navigation_styles
-    ),
-    server = function(input, output) {}
-  )
-}
-
-examples$Nav <- makeNav()
-
-
-# OverflowSet
-
-
-makeOverflowSet <- function() {
-  items <- list(
-    list(key = "item1", icon = "Add", name = "Link 1"),
-    list(key = "item2", icon = "Upload", name = "Link 2"),
-    list(key = "item3", icon = "Share", name = "Link 3")
-  )
-  overflowItems <- list(
-    list(key = "item4", icon = "Mail", name = "Overflow Link 1"),
-    list(key = "item5", icon = "Calendar", name = "Overflow Link 2")
-  )
-  onRenderItem <- JS("item =>
-    jsmodule['react'].createElement(jsmodule['@fluentui/react'].CommandBarButton, {
-      role: 'menuitem',
-      iconProps: { iconName: item.icon },
-      styles: {
-        root: { padding: '10px' }
-      }
-    })
-  ")
-  onRenderOverflowButton <- JS("overflowItems =>
-    jsmodule['react'].createElement(jsmodule['@fluentui/react'].CommandBarButton, {
-      role: 'menuitem',
-      title: 'More items',
-      styles: {
-        root: { padding: '10px' }
-      },
-      menuIconProps: { iconName: 'More' },
-      menuProps: { items: overflowItems }
+#
+# Read all the examples on the parent directory and create a list
+# where each element is a shiny dashboard with a 'ui´ and ´sever´ elements
+  buildExamples <- function(exampleFiles = NULL, excludePatterns = "Theme") {
```

Comment:
I think we'll be better off with an explicit list of examples here. Cons: requires manual updates. Pros: less complex, more flexible (easier to pick one of the examples when we have many, or split components into sections in the future).

Reply:
Yup, I agree.  I'm a sucker for automation, but in this case it's not worth it.